docs/howto/garbage-collection/standalone-gc.md

```diff
@@ -0,0 +1,189 @@
+```
+
+### Step 3: Download the docker image
+Download the image from the [lakefs-sgc](https://hub.docker.com/repository/docker/treeverse/lakefs-sgc/general) repository:
+```bash
+docker pull treeverse/lakefs-sgc:tagname
+```
+
+## Usage
+
+### AWS Credentials
+Currently, `lakefs-sgc` does not provide an option to explicitly set AWS credentials. It relies on the hosting machine
+to be set up correctly, and reads the AWS credentials from the machine.
+
+This means, you should set up your machine however AWS expects you to set it. \
+For example, by following their guide on [configuring the AWS CLI](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-configure.html).
+
+### Configuration
+The following configuration keys are available:
+
+| Key                        | Description                                                                                        | Default value                  | Possible values                                         |
+|----------------------------|----------------------------------------------------------------------------------------------------|--------------------------------|---------------------------------------------------------|
+| `logging.format`           | Logs output format                                                                                 | "text"                         | "text","json"                                           |
+| `logging.level`            | Logs level                                                                                         | "info"                         | "error","warn",info","debug","trace"                    |
+| `logging.output`           | Where to output the logs to                                                                        | "-"                            | "-" (stdout), "=" (stderr), or any string for file path |
+| `logging.file_max_size_mb` | Max file size for logs output (relevant only if `logging.output` is set to a file path)            | 102400 (100MiB)                | number                                                  |
+| `logging.files_keep`       | Number of files to keep for logs rotation (relevant only if `logging.output` is set to a file path | 100                            | number                                                  |
+| `cache_dir`                | Directory to use for caching data during run                                                       | ~/.lakefs-sgc/data             | string                                                  |
+| `aws.max_page_size`        | Max number of items per page when listing objects in AWS                                           | not set (AWS defaults to 1000) | number                                                  |
+| `objects_min_age`          | Ignore any object that is last modified within this time frame ("cutoff time")                     | "6h"                           | duration                                                |
```

Comment:
Add a warning that it could be dangerous to change this value to something shorter. It's hardcoded in the server and making less than 6h could cause ongoing writes to fail.

Reply:
Done